tailwind.config.js

```diff
@@ -1,4 +1,4 @@
 // Adds a JS shim to load the TS config at runtime, enabling
 // support for addons expecting tailwind.config.js.
-const jiti = require("jiti")(__dirname);
-module.exports = jiti("@plyaz/devtools/configs/tailwind.config").default;
+import tailwindConfig from '@plyaz/devtools/configs/tailwind.config';
+export default tailwindConfig;
```

Comment:
This way config doesn't work for `storybook-addon-tailwind-autodocs` and that's why, I found `jiti` library as a solution to fix this issue. For [reference](https://plyaz.slack.com/archives/C08SA1PKUQH/p1750715048082119)

Let me know your thoughts about it.

Reply:
Also, I'm checking build issues for this PR.

Reply:
Since it make sense to use `jiti` for fixing that issue, we can aim to use

Reply:
Reverted 🙌